operations/companies.md

```diff
@@ -114,6 +115,7 @@ Returns all company profiles of the enterprise, possibly filtered by identifiers
 | `AccountingCode` | string | optional | Accounting code of the company. |
 | `Address` | [Address](configuration.md#address) | optional | Address of the company \(if it is non-empty, otherwise `null`\). |
 | `BillingCode` | string | optional | Billing code of the company. |
+| `InvoicingEmail` | string | optional | Email for issuing invoices to the company. |
```

Comment:
Again, deprecated fields like `Address` should not be here, whilst `Telephone` or `Iata` etc are missing. Order of the fields also doesn't match the returned DTO.

Reply:
✅